tests/parent_models.py

```diff
@@ -38,6 +38,8 @@ def setUp(self):
 
     def tearDown(self):
         """This runs after each test"""
+        db.session.query(Inventory).delete()  # clean up the test
```

Comment:
These changes look good to me

In your sandbox, can you try doing the following:

1. Run honcho start
2. Run behave
3. Run green
4. Repeat steps 2 and 3 a few times in that order

I want to see if the green tests will ever hang. I attempted to fix this persistent data issue in my add_flask_restx branch and ended up breaking the green tests where, if I run behave first and then green, the green tests never run

I'm not sure what I did to break it, it may not have had to do with deleting the leftover data. I do not think the issue I described will happen in the Github Action workflow as the BDD and TDD tests run in their own containers

Reply:
If you can confirm that this behavior does not happen, then you can merge w/o my approval

Reply:
Tried repeating thrice and worked everytime!